Bottom: 0e54e133e108dc70734d354f620c0e50fbcd59ee
Top:    e9cde75c01604a0ed5900bc42f0ef98321c73fcf
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-19 11:25:29 +0800

Refresh of autofs-5.0.7-misc-man-page-fixes.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 3f9337f..997fd50 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -1734,6 +1734,7 @@ static void usage(void)
 		"			set daemon log verbosity\n"
 		"	-C --dont-check-daemon\n"
 		"			don't check if daemon is already running\n"
+		"	-F --force	forceably clean up all automounts at start\n"
 		"	-V --version	print version, build config and exit\n"
 		, program);
 }
